### DISCUSSION TITLE
UP base image

### DISCUSSION THREAD

**Discussion by SpectreKr** [hassio-addons/addon-nginx-proxy-manager#269] [Q&A]
Kind time of the day.
Is it possible to upgrade the base image to version 2.9.2. There are added new plugins for receiving ssl via dns.

**Comment by knirefel** [comment]
Any chance for an update to the latest version?

**Reply by SpectreKr** [reply]
If you tell me how this can be done in HASSIO, I will try to update.

**Comment by mxwi** [comment]
Is this add-on still alive? 

Since in most enviroments this proxy exposes several platforms to the internet, an update image is very important!

**Comment by frenck** [comment]
A new release is expected to become available this weekend.

../Frenck
